src/pykeen/nn/functional.py

```diff
@@ -125,14 +122,14 @@ def complex_interaction(
     .. math ::
         Re(\langle h, r, conj(t) \rangle)
 
-    :param h: shape: (batch_size, num_heads, 1, 1, `2*dim`)
+    :param h: shape: (*batch_dims, `2*dim`)
```

Comment:
unfortunately every place you add a `*` that's not inside some backticks, it interprets it as the opening of a section that should be in italics. See just to the right where we had to wrap `2*dim` in backticks to avoid this problem

Reply:
0c0b73e